ui/packages/consul-ui/app/routes/dc/services/show/index.js

```diff
@@ -11,29 +11,25 @@ export default Route.extend({
     // so check the length here.
     let to = 'topology';
     const parentModel = this.modelFor(parent);
-    const hasProxy = get(parentModel, 'proxies');
+    const hasProxy = get(parentModel, 'proxies').length !== 0;
```

Comment:
I'm still wondering about this one, maybe put the length inside the string argument for `get` which is safer ( `get(parentModel, 'proxies.length')` ). I'm not entirely sure by looking at this changeset if `proxies` is ever 'non-array-like'.